src/telemetry/TelemetryReporterProxy.ts

```diff
@@ -3,22 +3,41 @@
  *  Licensed under the MIT License. See License.md in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { ITelemetryReporter } from "vscode-azureextensionui";
-import { ITelemetryPublisher } from "./TelemetryPublisher";
+import { ITelemetryReporter } from 'vscode-azureextensionui';
+import { IExperimentationTelemetry } from 'vscode-tas-client';
+
+export class TelemetryReporterProxy implements ITelemetryReporter, IExperimentationTelemetry {
+    private readonly sharedProperties: { [key: string]: string } = {};
 
-export class TelemetryReporterProxy implements ITelemetryReporter {
     public constructor(
-        private readonly publisher: ITelemetryPublisher,
         private readonly wrappedReporter: ITelemetryReporter) {
     }
 
-    public sendTelemetryEvent(eventName: string, properties?: { [key: string]: string; }, measurements?: { [key: string]: number; }): void {
-        this.wrappedReporter.sendTelemetryEvent(eventName, properties, measurements);
+    public sendTelemetryErrorEvent(eventName: string, properties?: { [key: string]: string; }, measurements?: { [key: string]: number; }, errorProps?: string[]): void {
```

Comment:
I'm not sure I get this, why is it renamed as an error event? And what do `sharedProperties` represent?

Reply:
It's not an error, but `sendTelemetryEvent` was removed. I'm trying to work around [this issue](https://github.com/microsoft/vscode-azuretools/issues/693). It's probably going to change more.